src/scss/global.scss

```diff
@@ -180,6 +180,24 @@ button.btn {
     margin-left: auto;
     margin-right: auto;
   }
+  &.btn-primary {
```

Comment:
[nit]
```suggestion
  }

  &.btn-primary {
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1377/commits/de659957079dd74136f26d29e9b952be12add510